aspnetcore/signalr/supported-platforms.md

```diff
@@ -0,0 +1,37 @@
+---
+title: ASP.NET Core SignalR supported platforms
+author: rachelappel
+description: Supported platforms for ASP.NET Core SignalR 
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 04/20/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/supported-platforms
+---
+
+# ASP.NET Core SignalR supported platforms
+
+## Server system requirements
+
+SignalR for ASP.NET Core supports any server platform ASP.NET Core supports.
+
+## Client system requirements
+
+### Browser support
+
+The SignalR for ASP.NET Core JavaScript client supports the following browsers:
+
+| Browser | Version |
+| ------- | ------- |
+| Microsoft Internet Explorer | 11 |
+| Microsoft Edge | current |
+| Mozilla Firefox | current |
+| Google Chrome | current; includes Android |
```

Comment:
Let's put the "includes Android" and "includes iOS" in the Browser column actually, so "Google Chrome (including Android)" and "Safari (including iOS)"